xmake/rules/plugin/vsxmake/xmake.lua

```diff
@@ -37,7 +37,7 @@ rule("plugin.vsxmake.autoupdate")
         import("core.base.task")
 
         -- run only once for all xmake process in vs
-        local tmpfile = os.tmpfile(path.join(os.projectdir(), "plugin.vsxmake.autoupdate"))
+        local tmpfile = path.join(os.projectdir(), ".xmake", "plugin.vsxmake.autoupdate")
```

Comment:
please use local build/.gens direcrory

```lua
import("core.project.config")
local tmpfile = path.join(config.buildir(), ".gens", "rules", "plugin.vsxmake.autoupdate")
```

like this

https://github.com/xmake-io/xmake/blob/171780c824fdacef1173f6363c8c173b9906ab9d/xmake/core/project/target.lua#L890